### PR TITLE
Add thai language update missing translation document

### PR DESCRIPTION
**Pull request by jonoyeong**
## Description
Adding Thai language file.

## Specific Changes proposed
Please list the specific changes involved in this pull request.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [x] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

## Screenshots
Screen Shot 2020-11-13 at 11 09 15 AM
Screen Shot 2020-11-13 at 11 09 07 AM

### CODE REVIEW

**Comment by jonoyeong** [comment]
For some reason `var player = videojs(vid, {language: 'th'});` in my sandbox/index.html file is not changing the language of the video player. 

Should I instead change my browser/OS language?

**Comment by mister-ben** [comment]
Make sure you add the Thai translations script to index.html - `<script src="../dist/lang/th.js"></script>`.

**Review by mister-ben — APPROVED** [review]

**Comment by jonoyeong** [comment]
@mister-ben That worked like a charm thankyou! Added screenshots to the PR.

File: lang/th.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  "Duration": "ระยะเวลา",
+  "Remaining Time": "เวลาที่เหลือ",
+  "Stream Type": "ประเภทของสตรีม",
+  "LIVE": "ถ่ายทอดสด",
+  "Seek to live, currently behind live": "หาโอกาสที่จะถ่ายทอดสด กำลังอยู่เบื้องหลังการถ่ายทอดสดในขณะนี้",
+  "Seek to live, currently playing live": "หาโอกาสที่จะถ่ายทอดสด กำลังเล่นแบบสดในขณะนี้",
+  "Loaded": "โหลดแล้ว",
+  "Progress": "ความคืบหน้า",
+  "Progress Bar": "แถบแสดงความคืบหน้า",
+  "progress bar timing: currentTime={1} duration={2}": "{1} ของ {2}",
```
Comment (gkatsev) [inline lang/th.json:17]: thanks for getting the proper translation here!

**Review by gkatsev — APPROVED** [review]

**Pull request merged**